### PR TITLE
feat(poste-io): instructions and customization

### LINKED ISSUE #220: Poste.io not running
I tried to install Poste.io but after the install, no logs whatsoever are showing?

the domain name is something like this: mail.mydomain.com

any ideas on how to find out the reason ?

thank you

### PR DESCRIPTION
**Pull request by ronaldloyko**
This pull request updates the one-click-app for [Poste.io](https://poste.io/).

## Changes

- Fixed TLS certificate issues
- Added instructions
  - Added pre-deploy script
  - Explained how to set up HTTPS correctly
  - Added guidance notes on how to set up DNS entries
- Added more variables for customization (e.g. disabling ClamAV, Webmail and Rspamd)

## Checks

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

## References

- closes #220

### CODE REVIEW

File: public/v4/apps/poste-io.yml [hunk 1]
```diff
@@ -1,52 +1,143 @@
...
+        - id: $$cap_POSTE_HTTPS
+          label: Networking | Use HTTPS
+          description: >-
+              Whether to redirect to encrypted HTTP.
+              It's recommended to leave it to `OFF` since CapRover acts as reverse proxy and handles HTTPS.
+              If you want to enable it, remove the value and leave it blank and then to change the configuration and ports accordingly.
+          defaultValue: 'OFF'
+        - id: $$cap_POSTE_PORT_SMTP_UNENCRYPTED
+          label: Networking | Unencrypted SMTP Port
+          description: Port for unencrypted mail submission. Should not be changed.
```
Comment (githubsaturn) [inline public/v4/apps/poste-io.yml:106]: If they should not be changed, is there any benefit in keeping them as variables? Why not hardcode them for the sake of simplicity of the file and future maintenance?

Comment (ronaldloyko) [reply public/v4/apps/poste-io.yml]: Agree, reverted to hardcoded values.
Thanks.

**Comment by githubsaturn** [comment]
Thank you!

**Comment by mredodos** [comment]
@ronaldloyko hi i have problem with instruction and caprover functionality

**Pull request merged**